src/plugins/workspace/public/utils.ts

```diff
@@ -539,6 +539,29 @@ export const fetchDataSourceConnections = async (
   }
 };
 
+export const fetchDirectQueryConnectionsByIDs = async (
```

Comment:
I guess we need a unit test for this function.

Reply:
okok, I will add one